src/routes/landing/components/FilterableCitySelector.js

```diff
@@ -17,34 +19,70 @@ type PropsType = {
 }
 
 type StateType = {
-  filterText: string
+  filterText: string,
+  currentLongitude: number | null,
+  currentLatitude: number | null,
+  renderLocationList: boolean
 }
 
 class FilterableCitySelector extends React.Component<PropsType, StateType> {
   constructor (props: PropsType) {
     super(props)
-    this.state = { filterText: '' }
+    this.state = {
+      filterText: '',
+      currentLongitude: null,
+      currentLatitude: null,
+      renderLocationList: false
+    }
   }
 
   onFilterTextChange = (filterText: string) => this.setState({ filterText })
 
+  _onPressLocationButton = () => {
+    if (Platform.OS === 'ios') {
+      Geolocation.requestAuthorization()
+    }
+    Geolocation.getCurrentPosition(
+      position => {
+        const currentLongitude = position.coords.longitude
+        const currentLatitude = position.coords.latitude
+        this.setState({ currentLongitude: currentLongitude })
+        this.setState({ currentLatitude: currentLatitude })
+      },
+      error => {
+        alert(error.message)
+      },
+      { enableHighAccuracy: false, timeout: 20000, maximumAge: 1000 }
+    )
+    this.setState({ renderLocationList: true })
+  }
+
   render () {
     const { cities, t, theme } = this.props
     const filterText = this.state.filterText
 
     return (
       <>
-        <SearchInput
+        <View style={{ flexDirection: 'row', justifyContent: 'space-between' }} >
```

Comment:
You can use a styled component for this:
something like:
```
const MyView = styled.View`
flexDirection: row; 
justifyContent: space-between`
```
(with a better name ofc). And then just use `<MyView></MyView>` instead.